src/Services/helper.ts

```diff
@@ -0,0 +1,16 @@
+import {CAMERAS} from "../Controllers/RoverController";
+import {Photo} from "../Models/RoverModels";
+
+export function trim_photos_json(data: any): Array<Photo> {

```

Comment:
`data: any` - ideally we'd have the exact type

also looking at the code, you could have passed `data.photos` directly